src/renderer/components/editorWithTabs/editor.vue

```diff
@@ -423,6 +424,7 @@
           if (history) {
             editor.setHistory(history)
           }
+          console.log(cursor) // #DEBUG
```

Comment:
⚙️ no debug codes, I think we should add `eslint rules` or some other lint before git push to prevent debug codes.